### ISSUE TITLE
Changed to 8.0.1 now cannot find packages in autogenerated classes

### ISSUE THREAD

**Issue by stillie** [JakeWharton/butterknife#574]
Hi Jake,

I have just changed to 8.0.1 and made all the changes needed such as @Bind to @BindView as well as the unbinder.unbind() etc.

I have a Package that has my registration Classes in and this is the file structure:

registration <- package
-------- otp <- package
-------- StepOne <- package
---------------- StepOneFragment <- Class 
---------------- StepOneActivity <- Class
-------- StepTwo <- package
---------------- StepTwoFragment <- Class
---------------- StepTwoActivity <- Class
-------- StepThree <- package
---------------- StepThreeFragment <- Class
---------------- StepThreeActivity <- Class

The issue I am having is the autogenerated classes that are Generated by butterknife cannot find the StepOne, StepTwo and StepThree packages, only after the change from ButterKnife 7 - 8

If i move all the classes out of the StepOne package and into the registration package, it works fine, its when it is that deep in the file structure

Hope to hear from you soon

**Comment by stillie** [comment]
I just found the issue!!! Like just after sending this! 
Please see this Stackoverflow post 

http://stackoverflow.com/questions/36940575/cannot-find-symbol-class-on-butter-knife-generated-code

Its because the package had Upper case in, the minute i changed it to stepone, steptwo and stepthree it worked fine.

Thanks

**Comment by JakeWharton** [comment]
You are including the butterknife-compiler per the readme, right?

**Comment by stillie** [comment]
Yea it is included, please see my comment

**Comment by JakeWharton** [comment]
This will be fixed in the next release. There's a separate issue tracking
it which has already been closed.

**Comment by stkent** [comment]
See https://github.com/JakeWharton/butterknife/issues/507
